client/src/app/site/users/components/user-detail/user-detail.component.ts

```diff
@@ -138,18 +141,22 @@ export class UserDetailComponent extends BaseModelContextComponent implements On
             this.newUser = true;
             this.setEditMode(true);
         } else {
-            this.route.params.subscribe(params => {
-                this.loadUserById(Number(params.id));
-            });
+            this.subscriptions.push(
+                this.route.params.subscribe(params => {
+                    this.userId = Number(params.id);
```

Comment:
`Number` is different from `number` -> use `parseInt()` or just `+params.id` (my fav;) )